src/NuGetPackageVerifier/Program.cs

```diff
@@ -78,6 +78,7 @@ public static int Main(string[] args)
                 new RequiredPackageMetadataRule(),
                 new SatellitePackageRule(),
                 new StrictSemanticVersionValidationRule(),
+                new AssemblyHasCorrectJsonNetVersionRule(),
```

Comment:
Will address these comments as part of https://github.com/aspnet/DnxTools/issues/20. :smile: